core/src/nucleus/ribosome/run_dna.rs

```diff
@@ -27,7 +27,10 @@ pub fn run_dna(
     parameters: Option<Vec<u8>>,
 ) -> ZomeFnResult {
     // Create wasm module from wasm binary
-    let module = wasmi::Module::from_buffer(wasm).expect("wasm binary should be valid");
+    let module = match wasmi::Module::from_buffer(wasm) {
+        Ok(module) => module,
+        Err(e) => return Err(HolochainError::ErrorGeneric(e.into())),
+    };
```

Comment:
You can use the `?` operator here

Reply:
Please don't stall merges with change requests like this. We agreed that our etiquette suggests for pushing a commit to fix those kind of issues ourselves.

Reply:
done

Reply:
please confirm and hit merge, @sphinxc0re